src/Microsoft.Health.Fhir.Core/Features/Search/Filters/MissingDataFilterCriteria.cs

```diff
@@ -0,0 +1,71 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using EnsureThat;
+using Microsoft.Health.Fhir.Core.Features.Persistence;
+
+namespace Microsoft.Health.Fhir.Core.Features.Search.Filters
+{
+    /// <summary>
+    /// Following US Core requirements, <see cref="MissingDataFilterCriteria"/> filters out resources with missing status codes.
+    /// Reference: http://hl7.org/fhir/us/core/STU3.1.1/general-guidance.html#missing-data
```

Comment:
Do we need to be USCore version aware at some point?

Reply:
That's a good question.
Let me check.

Reply:
As discussed last week, we are not handling different versions of USCore today.